packages/bezier-vscode/src/server.ts

```diff
@@ -11,21 +12,54 @@ import {
 } from 'vscode-languageserver/node'
 import { TextDocument } from 'vscode-languageserver-textdocument'
 
-import { hexToRGBA } from './utils'
+import { deepMerge, hexToRGBA } from './utils'
 
-const tokens = {
-  ..._tokens.lightTheme,
-  ..._tokens.global,
-  color: {
-    ..._tokens.lightTheme.color,
-    ..._tokens.global.color,
-  },
+type TokenValue = string | number | Record<string, string | number>
+type TokenMap = Record<string, Record<string, TokenValue>>
+
+const assignToTokenMap = (
+  target: TokenMap,
+  source: TokenMap,
+  fn: Function = (v: TokenMap) => v
+) => {
+  Object.entries(source).forEach(([category, tokenObject]) => {
+    if (target[category] === undefined) {
+      target[category] = {}
+    }
+    Object.entries(tokenObject).forEach(([name, value]) => {
+      target[category][name] = fn(value)
+    })
+  })
 }
```

Comment:
```suggestion
type TokenValue = string | number
type AlphaTokenValue = Record<string, TokenValue>
type TokenMap = Record<string, Record<string, TokenValue | AlphaTokenValue>>

const assignToTokenMap = (
  target: TokenMap,
  source: TokenMap,
  transformValue: (v: TokenValue | AlphaTokenValue) => TokenValue = (v) =>
    v as TokenValue
) => {
  Object.entries(source).forEach(([category, tokenObject]) => {
    if (target[category] === undefined) {
      target[category] = {}
    }
    Object.entries(tokenObject).forEach(([name, value]) => {
      target[category][name] = transformValue(value)
    })
  })
```

버전 별 토큰에 alias를 적절히 붙이고, 타입 단언을 사용하더라도 의도를 더 드러내는 편이 좋을 거 같습니다!